src/psyclone/tests/psyir/nodes/call_test.py

```diff
@@ -180,7 +180,7 @@ def test_call_create_error3():
         _ = Call.create(
```

Comment:
Not the fault of this PR, but `pylint` says:
```
call_test.py:64:11: C1803: 'call.children == []' can be simplified to 'not call.children' as an empty sequence is falsey (use-implicit-booleaness-not-comparison)
call_test.py:71:11: C1803: 'call.children == []' can be simplified to 'not call.children' as an empty sequence is falsey (use-implicit-booleaness-not-comparison)
```

Would you mind improving this?

Reply:
pylint should be run from the `tests` directory for test files. That way, this message is suppressed.